modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/BaseUADMVCActionCommand.java

```diff
@@ -29,16 +28,16 @@
  */
 public abstract class BaseUADMVCActionCommand extends BaseMVCActionCommand {
 
-	protected UADEntity getUADEntity(
+	protected Object getUADEntity(
```

Comment:
I think we should rename this to `getEntity` to avoid confusion with the UADEntity.